class15/hhi-5258/HHiApplication/app/src/main/java/com/hhi/myapplication/MainActivity.kt

```diff
@@ -2,10 +2,56 @@ package com.hhi.myapplication
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.hhi.myapplication.api.MovieData
+import com.hhi.myapplication.api.NaverAPI
+import kotlinx.android.synthetic.main.activity_main.*
+import retrofit2.Call
+import retrofit2.Response
 
 class MainActivity : AppCompatActivity() {
+    private val recyclerAdapter = RecyclerAdapter()
+    private val api = NaverAPI.create()
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        initView()
+    }
+
+    private fun initView() {
+        main_recyclerview.layoutManager =
+            LinearLayoutManager(this, LinearLayoutManager.VERTICAL, false)
+        main_recyclerview.setHasFixedSize(false)
+        main_recyclerview.adapter = recyclerAdapter
+        main_btn_search.setOnClickListener {
+            if (main_edit_search.text.isEmpty()) {
+                return@setOnClickListener
+            } else {
+                searchMovie(main_edit_search.text.toString())
```

Comment:
이렇게 else문을 제거하는 것도 괜찮을 것 같아요!
```suggestion
            if (main_edit_search.text.isNotEmpty()) {
                searchMovie(main_edit_search.text.toString())
            }
```